src/server/status/server_status.js

```diff
@@ -2,15 +2,24 @@ import _ from 'lodash';
 
 import states from './states';
 import Status from './status';
+import PluginStatus from './plugin_status';
 
 module.exports = class ServerStatus {
   constructor(server) {
     this.server = server;
     this._created = {};
   }
 
-  create(plugin) {
-    return (this._created[plugin.id] = new Status(plugin, this.server));
+  create(id) {
```

Comment:
This PR creates two types of statuses: plugin status and service status. Since we have a method called `createForPlugin` for attaching a new Plugin status, should we rename the `create` method to `createForService` for consistency?